docs/getting-started.md

```diff
@@ -4,39 +4,41 @@ category: 5d4c940cae4e610060475769
 ---
 
 <p align="center">
-  <img width="350" src="./images/logo.svg" />
```

Comment:
Most of the changes in this file are just to get it up to par with the main `README.md` file, but I think we should do some reuse if possible. See #486